cmd/clientd/internal/sync/embed/copy-and-truncate.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/ash
```

Comment:
Are we truely using `ash`? If not I have feedback for this file, but if we are I don't

Reply:
I've only tested this on alpine rn - file locking works different on ash/bash AFAIK. The containers I'm planning on running these on are alpine based but idk - I could definitely be swayed. Bash is more portable overall